### DISCUSSION TITLE
Version Rollback

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#25066] [Q&A]
### Type

Game behaviour

### Bug description

I don't know, but when I opened 'osulazer' today, it rolled back from 924 to 815 instead of updating to 1008. I don't know what happened, but I can think of something related to it. I previously copied the 'app 2023.924.1' from the osulazer on another computer to 'osulazer' on this computer, and it automatically used version 924. However, now that the new version is released, it has rolled back to version 815.

After checking for updates, it shows that 815 is the latest version.

I noticed that 'Your local database is too new to work with this version of osu!. Please close osu! And install the latest release to recover your data.' I believe it is the result of its rollback version.

So, I tried to download the nupkg file to update the version, but it failed. osu! did not detect it, and the latest version recognized now is 924

I checked the log and found that the problem may be due to my network connection, but I can't find the reason.

Okay, I made a dangerous move. I unzipped nupkg and put it in osulazer, but it succeeded. It has now been updated to 1008

https://github.com/ppy/osu/assets/87159802/3a5e0148-edc9-4956-a108-711dedfc7a83

### Version

2023.924.1

### Logs

[network.log](https://github.com/ppy/osu/files/12840721/network.log)
[performance.log](https://github.com/ppy/osu/files/12840722/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12840723/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12840724/updater.log)
[database.log](https://github.com/ppy/osu/files/12840725/database.log)
[input.log](https://github.com/ppy/osu/files/12840727/input.log)

**Comment by bdach** [comment]
Please attach the installation log from `C:\Users\[username]\AppData\Local\SquirrelTemp\SquirrelSetup.log`.

**Reply by nana123126** [reply]
The folder 'SquirrelTemp' does not exist in 'C:\Users\[username]\AppData\Local ' (Any user)

**Reply by bdach** [reply]
I'm not really sure where to even begin then. Logs provided in the OP look clean.

**Reply by nana123126** [reply]
I noticed some things before reinstalling the Lazer. I have two folders containing the complete osulazer game, and I only ran one of them. However, when it was updated, the updated files were placed in another game folder. The problem may be because the name of the other game folder is 'osulazer' ?
